db/migrate/20201110090452_create_drivers.rb

```diff
@@ -1,13 +1,13 @@
 class CreateDrivers < ActiveRecord::Migration[6.0]
   def change
     create_table :drivers do |t|
-      t.string :name
-      t.string :license_number
-      t.string :mobile_number
-      t.string :email_address
-      t.integer :age
+      t.string :name, null: false
+      t.string :license_number, null: false
+      t.string :mobile_number, null: false
+      t.string :email_address, null: false
+      t.integer :age, null: false
       t.text :notes
-      t.boolean :status
+      t.boolean :status, null: false
```

Comment:
remember that after making this change you need to re-run the migration or the schema wont be updated